pkgs/tools/misc/zellij/default.nix

```diff
@@ -8,20 +8,22 @@
 , openssl
 , zellij
 , testVersion
+, DiskArbitration
+, Foundation
```

Comment:
Some minor nitpick. We sort these according to usage. Tests comes after build, so `testVersion` and `zellij` are at the end.

Reply:
Could you amend these changes into the "zellij: Fix building on Darwin platform" commit before I merge it?